plugins/inputs/bond/bond_test.go

```diff
@@ -7,34 +7,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var sampleTest802 = `
-Ethernet Channel Bonding Driver: v3.5.0 (November 4, 2008)
-
-Bonding Mode: IEEE 802.3ad Dynamic link aggregation
-Transmit Hash Policy: layer2 (0)
-MII Status: up
-MII Polling Interval (ms): 100
-Up Delay (ms): 0
-Down Delay (ms): 0
-
-802.3ad info
-LACP rate: fast
-Aggregator selection policy (ad_select): stable
-bond bond0 has no active aggregator
-
-Slave Interface: eth1
-MII Status: up
-Link Failure Count: 0
-Permanent HW addr: 00:0c:29:f5:b7:11
-Aggregator ID: N/A
-
-Slave Interface: eth2
-MII Status: up
-Link Failure Count: 3
-Permanent HW addr: 00:0c:29:f5:b7:1b
-Aggregator ID: N/A
-`
-
```

Comment:
Please do not remove existing tests to ensure we are still covering the current uses. Instead just add a new test-case.

Reply:
In order to get these tests working _at all_ I had to remove that test.